src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -39,5 +39,29 @@ private unsafe string DisplayString
                 }
             }
         }
+
+        public Vector64<byte> AsByte() => AsByte();
+
+        public Vector64<double> AsDouble() => AsDouble();
+
+        public Vector64<short> AsInt16() => AsInt16();
+
+        public Vector64<int> AsInt32() => AsInt32();
+
+        public Vector64<long> AsInt64() => AsInt64();
+
+        [CLSCompliant(false)]
+        public Vector64<sbyte> AsSByte() => AsSByte();
+
+        public Vector64<float> AsSingle() => AsSingle();
+
+        [CLSCompliant(false)]
+        public Vector64<ushort> AsUInt16() => AsUInt16();
+
+        [CLSCompliant(false)]
+        public Vector64<uint> AsUInt32() => AsUInt32();
+
+        [CLSCompliant(false)]
+        public Vector64<ulong> AsUInt64() => AsUInt64();
```

Comment:
Should they be extension methods?

Reply:
Why would they need to be extension methods, rather than directly exposing on the type?

Reply:
Oh, never mind. They are in `struct`.